src/components/GoogleCalendarButton.js

```diff
@@ -26,41 +26,90 @@ export default function GoogleCalendarButton() {
     Mo: '2020-04-06T',
   };
   const events = [];
-  const handleClick = () => {
-    sections.forEach(
-      (section) => {
-        // '2015-05-28T09:00:00-07:00'
-        const dow = section.event.dow;
-        const date = dateOfTheFirstClass[dow];
-        const startTime = `${date + section.event.start.hour}:${section.event.start.minute}:00${timeZone}`;
-        const endTime = `${date + section.event.end.hour}:${section.event.end.minute}:00${timeZone}`;
-        const event = {
-          summary: `${section.subjectId}${section.courseId}`,
-          description: section.name,
-          location: section.event.location,
-          start: {
-            dateTime: startTime,
-            timeZone: 'America/Chicago',
-          },
-          end: {
-            dateTime: endTime,
-            timeZone: 'America/Chicago',
+  const clientID = '4436581685-mcdkhnh50a6di21k15do7d4a2orq7faf.apps.googleusercontent.com'
+  // const clientSecret = 'gPwoF99zNeSrs_JXcVaNATub'
+  const APIKey = 'AIzaSyCssknAiVBtWhHE3O1wjVte829dxoxQ-pM'
+
+  sections.forEach(
+    (section) => {
+      // '2015-05-28T09:00:00-07:00'
+      const dow = section.event.dow;
+      const date = dateOfTheFirstClass[dow];
+      const startTime = `${date + section.event.start.hour}:${section.event.start.minute}:00`;
+      const endTime = `${date + section.event.end.hour}:${section.event.end.minute}:00`;
+      const event = {
+        summary: `${section.subjectId} ${section.courseId}`,
+        description: section.name,
+        location: section.event.location,
+        start: {
+          dateTime: startTime,
+          timeZone: 'America/Chicago',
+        },
+        end: {
+          dateTime: endTime,
+          timeZone: 'America/Chicago',
+        },
+        recurrence: [
+          'RRULE:FREQ=WEEKLY;COUNT=10',
+        ],
+      };
+      events.push(event);
+    },
+  );
+
+  const initAuth = () => {
+    window.gapi.load('auth2', () => {
+      window.gapi.auth2.init({
+        client_id: clientID,
+      }).then(() => {
```

Comment:
Let's try to use `async`/`await` syntax for promises (goes for all other instances in this file)

Reply:
Can you explain a bit more about what you mean here? I am a little confused about where to put the async/await keywords in this case

Reply:
async await is just a different way to write code that deals with Promises. Feel free to read up on it online yourself if you want, but you and Kevin or me should touch base on this on slack/at a meeting.